### PR TITLE
924195-Add-Calendar-UG-Added UG for Calendar

### PR DESCRIPTION
**Pull request by HadayaRahman-SF4673**
Description:
Added Calendar Ug to Toolkit Repository

### CODE REVIEW

File: maui-toolkit-toc.html [hunk 1]
```diff
@@ -16,6 +16,25 @@
         </ul>
     </li>
 	<li><a href="/maui-toolkit/Migration">Migrate from Syncfusion .NET MAUI</a></li>
+    <li>
+        SfCalendar
+        <ul>
+            <li><a href="/maui-toolkit/Calendar/overview">Overview</a></li>
+            <li><a href="/maui-toolkit/Calendar/getting-started">Getting Started</a></li>
+            <li><a href="/maui-toolkit/Calendar/migration">Migrate from Xamarin.Forms</a></li>
```
Comment (Jeyasri-Murugan) [inline maui-toolkit-toc.html:24]: Remove this Migration file as not needed , remove file and link

File: maui-toolkit/Calendar/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Calendar Control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline maui-toolkit/Calendar/accessibility.md:3]: Update details in this
https://help.syncfusion.com/maui-toolkit/migration

Comment (HadayaRahman-SF4673) [reply maui-toolkit/Calendar/accessibility.md:3]: Updated It in Latest Commit

File: maui-toolkit-toc.html (hunk 1 above)
Comment (HadayaRahman-SF4673) [reply maui-toolkit-toc.html]: Removed the Respective Files and Link with Latest Commit

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**